packages/event-handler/src/types/appsync-graphql.ts

```diff
@@ -0,0 +1,139 @@
+import type { RouteHandlerRegistry } from '../appsync-graphql/RouteHandlerRegistry.js';
+
+// biome-ignore lint/suspicious/noExplicitAny: We intentionally use `any` here to represent any type of data and keep the logger is as flexible as possible.
+type Anything = any;
+
+/**
+ * Interface for a generic logger object.
+ */
+type GenericLogger = {
```

Comment:
You might've branched out before this was merged, but we moved this type and the one above (`Anything`) to a separate file so we can share it (see [here](https://github.com/aws-powertools/powertools-lambda-typescript/blob/main/packages/event-handler/src/types/common.ts)).

If possible let's use them instead.